enforcer_interface.go

```diff
@@ -58,6 +60,9 @@ type IEnforcer interface {
 	EnforceWithMatcher(matcher string, rvals ...interface{}) (bool, error)
 	EnforceEx(rvals ...interface{}) (bool, [][]string, error)
 	EnforceExWithMatcher(matcher string, rvals ...interface{}) (bool, [][]string, error)
+	IsAudoLoadRunning() bool
+	StartAutoLoadPolicy(d time.Duration)
```

Comment:
Maybe extract methods in SyncedInterface and embed it? Currently, IEnforcer is really huge.